docs/README.md

```diff
@@ -131,7 +130,7 @@ _Will enable brew and steam temperature control as well as pump control based on
 * [C-M4 screw K-Type thermocouple sensor](https://www.aliexpress.com/item/1005004948080451.html)
 * [40DA SSR Relay](https://www.aliexpress.com/item/4000045425145.html)
 * [Heat-resistant Silicone Wire](https://bit.ly/3tjSQbI)
-  * **18AWG - 1m:** Black, Red, White
+  * **18AWG - 2m:** Black, Red, White
```

Comment:
I tool the liberty to update that, can remove if needed. I found 1m to be barely enough as I needed to use some of it to bypass the PCB. I feel like adding 1m won't hurt anyone, even those that don't need it.